ProyectoIntegrAccion.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "Proyecto IntegrAcción",
+  "type": "comunidad",
+  "bio": "un proyecto de cooperación entre la Universidad Católica San Pablo y ACNUR"
+{
```

There us a missing `links` key. Please make sure you follow the [quickstart guide](https://linkfree.io/docs/quickstart). 

```suggestion
"links": [
    {
```